ui/src/App.js

```diff
@@ -1,15 +1,32 @@
-import Dashboard from "./Components/Dashboard/Dashboard";
-import { FEATURE_FLAGS, FEATURE_FLAGS_HEADERS} from "./Mock/featureFlags";
+import { useState } from 'react';
+import Dashboard from './Components/Dashboard/Dashboard';
+import Filters from './Components/Filters/Filters';
+import Search from './Components/Search/Search';
+import { useFilterContext } from './Context/filter_context';
+import { FEATURE_FLAGS, FEATURE_FLAGS_HEADERS } from './Mock/featureFlags';
 
 function App() {
+  const { filteredList, isLoading, featureFlagHeader, clearFilter } =
```

Comment:
Should `isLoading` be a part of `FilterContext`?

Reply:
It should be part of context where we are calling API